### DISCUSSION TITLE
Does it support .NET 7 ?

### DISCUSSION THREAD

**Discussion by SURYASPR** [rabbitmq/rabbitmq-dotnet-client#1330] [General]
### Is your feature request related to a problem? Please describe.

I see that RabbitMQ.Client has dependencies on .NET Framework 4.6, but I don't see any package which has dependency on .NET 7. Does that mean this nuget package doesn't support .net 7 ?

### Describe the solution you'd like

If this doesn't support .net 7, are there plans to support .net 7 in future ? Are there any other alternative packages which support .net 7 ?

**Comment by michaelklishin** [comment]
This client does support .NET 6 and 7.

**Reply by SURYASPR** [reply]
Oh great, I had a little confusion as .NET 7 is not listed in the Compatible target frameworks here https://www.nuget.org/packages/RabbitMQ.Client

**Reply by lukebakken** [reply]
Please refer to the `netstandard2.0` compatibility matrix:

https://learn.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0

**Reply by SURYASPR** [reply]
Okay Thanks for the info @michaelklishin and @lukebakken
